doozerlib/exectools.py

```diff
@@ -83,7 +87,7 @@ def cmd_assert(cmd, retries=1, pollrate=60, on_retry=None, set_env=None):
     return stdout, stderr
 
 
-def cmd_gather(cmd, set_env=None):
+def cmd_gather(cmd, set_env=None, realtime=False):
```

Comment:
No docstring for this boolean kwarg. A little exposition in the docstring would be nice.

Reply:
Done.